src/main/resources/templates/login.html

```diff
@@ -2,6 +2,9 @@
 <html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="../static/css/style.css" th:href="@{/css/style.css}" rel="stylesheet" />
```

Comment:
https://stackoverflow.com/questions/42774564/difference-between-thhref-and-href-when-linking-absolute-urls-in-thymeleaf so th:href is not necessary.